### ISSUE TITLE
ignition-common and gazebo bottles broken by ffmpeg version 4

### ISSUE THREAD

**Issue by scpeters** [osrf/homebrew-simulation#404]
ffmpeg version 4 got merged in https://github.com/Homebrew/homebrew-core/pull/26882 and it's broken the ignition-common and gazebo bottles:

* https://build.osrfoundation.org/view/os_homebrew/job/gazebo-install-one_liner-homebrew-amd64/886/console
* https://build.osrfoundation.org/view/os_homebrew/job/ignition_common-ci-ign-common1-homebrew-amd64/9/console

We need to fix the compilation errors and rebuild the bottles

~~~
ign-common/src/Video.cc:167:29:
 error: use of undeclared identifier 'CODEC_CAP_TRUNCATED'
  if (codec->capabilities & CODEC_CAP_TRUNCATED)
                            ^
ign-common/src/Video.cc:168:39:
 error: use of undeclared identifier 'CODEC_FLAG_TRUNCATED'
    this->dataPtr->codecCtx->flags |= CODEC_FLAG_TRUNCATED;
                                      ^
2 errors generated.
~~~

**Comment by scpeters** [comment]
I think the symbol is now called `AV_CODEC_CAP_TRUNCATED`

**Comment by scpeters** [comment]
fixed by #405
